generator/generator.yml

```diff
@@ -355,3 +355,40 @@ modules:
       - picoMobileMIB
       - picoIPv4MIB
       - picoIPv6MIB
+# Raritan MIB
+#
+# Used by Legran/Raritan PDUs
+#
+# http://support.raritan.com/px2/version-2.4.1/mibs/pdu2-mib-020400-39592.txt
+  raritan_pdu:
+    walk:
+      - 1.3.6.1.4.1.13742.6.5.2.4.1.4.1.1 # input voltage/current etc...  mono/tri phase(s)
```

Comment:
Please follow the existing pattern here

Reply:
You prefers have the oid like : PDU2-MIB::measurementsOutletSensorValue.$reglette_num.1.1 ?

Reply:
The name of the object would be usual.

Reply:
Ok no worry. I don't have time today. But tomorrow i change and test